records_mover/records/validated_records_hints.py

```diff
@@ -31,6 +76,9 @@ class ValidatedRecordsHints(NamedTuple):
     @staticmethod
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker